Formula/aescrypt.rb

```diff
@@ -4,6 +4,11 @@ class Aescrypt < Formula
   url "https://aescrypt.sourceforge.io/aescrypt-0.7.tar.gz"
   sha256 "7b17656cbbd76700d313a1c36824a197dfb776cadcbf3a748da5ee3d0791b92d"
 
+  livecheck do
+    url :homepage
+    regex(/aescrypt-(\d\.\d).t/i)
```

Comment:
```suggestion
    regex(/aescrypt[._-]v?(\d+(?:\.\d+)+)\.t/i)
```
Let's use something more flexible here.